DP02_11_User_Packages/DP02_11_Working_with_user_packages.ipynb

```diff
@@ -7,9 +7,9 @@
    "source": [
```

Comment:
<p>This log-out-and-back-in step is not necessary. The user just has to restart the NB: e.g., clear outputs, save, close and shutdown NB, then reopen it. The .user_setups file gets read, and astroML can be imported.</p><p> </p><p>Can you double check this and update the instructions here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Done</p>